.gitignore

```diff
@@ -90,3 +90,4 @@ NuGetExeBuild/
 NuGetRefBuild/
 Packages/
 NuGetTaskBuild/
+build/
```

Comment:
@JakeGinnivan @gep13 I honestly don't see the problem with my proposed solution. The files we want to track inside the `build` folder are already tracked by Git and won't stop being tracked just because we start ignoring the `build` folder. If we want to ensure that it never happens in the future, we can explicitly un-ignore the files with `!`.